docs/6.x/pack.md

```diff
@@ -912,3 +912,36 @@ When building a Cluster Image, `tele build` will discover `custom-planet:1.0.0`
 Docker image and vendor it along with other dependencies. During the Cluster
 installation all nodes with the role `worker` will use the custom "planet" Docker
 image instead of the default one.
+
+## Direct Upgrades From Older LTS Versions
+
+Kubernetes has a strict version skew [support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-version-skew) which makes
+cluster upgrades to versions of Kubernetes being more than a couple of minor versions apart complicated if not impossible, hence
+upgrading such a cluster to the version of Kubernetes that would break the version skew policy is not recommended. This can become a big issue if
+clusters are not kept up-to-date.
+
+To correctly upgrade such a cluster, it first needs to be upgraded to a cluster image with a compatible version of Kubernetes (according to the version
+skew policy). This step should be repeated until the cluster has been upgraded to the target version.
+
+For example, to upgrade a cluster based on Gravity `5.0.35` (which is based on Kubernetes `1.9.13`) to a cluster based on Gravity `5.5.20` (which
```

Comment:
This is a handy paragraph, I wonder if we should also put Gravity Version & K8 version in a table for easier scanning for customers.